tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -15,66 +15,128 @@
           preparsing_validators=[
               validators.hasLength(71),
           ],
-          postparsing_validators=[],
+          postparsing_validators=[
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.notMatches(1),
+                  result_field='SSN', result_function=validators.isNumber()
+                  ),
+              validators.validate__FAM_AFF__SSN(),
```

Comment:
curious if you played with the `and_validator` introduced in #2518 
https://github.com/raft-tech/TANF-app/blob/4b70e6aa36b0cc605579d38e6eabf679fe3811b7/tdrs-backend/tdpservice/parsers/validators.py#L22
i think that may give us enough flexibility to move away from custom validator functions in most cases. open to opinions though as i'm sure the expressions could get gnarly. i'd be a fan of at least utilizing the `make_validator` from the custom function so that the functions are consistent.